queries/highlights.scm

```diff
@@ -1,132 +1,271 @@
-(comment) @comment
+; Includes
 
 [
-  "("
-  ")"
-  "{"
-  "}"
-  "["
-  "]"
-] @punctuation.bracket
+  "use"
+] @include
 
-[
-  ";"
-  "."
-  ","
-] @punctuation.delimiter
+; Keywords
 
 [
-  "if"
-  "ifdef"
-  "then"
-  "else"
-  "elseif"
-  "end"
-  "try"
-  "while"
-  "for"
-  "use"
-  "as"
-  "var"
-  "let"
-  "embed"
-  "fun"
-  "be"
-  "new"
+  "type"
   "actor"
   "class"
-  "struct"
   "primitive"
   "interface"
   "trait"
-  "type"
+  "struct"
+  "embed"
+  "let"
+  "var"
+  "be"
+  (compile_intrinsic)
+  "as"
+  "consume"
+  "object"
+  "where"
+] @keyword
+
+[
+  "fun"
+] @keyword.function
+
+[
+  "new"
```

Comment:
this is marking the beginning of a constructor. It is never used as an operator. So I'd argue for `keyword.function` here.

Reply:
I applied similar JavaScript convention (`const a = new A()`) where A is highlighted as constructor, but new `@keyword.operator`, so I think it makes sense again here

Reply:
In javascript you create objects with `new`, so there it makes sense as an operator. In pony, `new` is only a keyword to mark the start of a constructor definition. So it is not an operator at all. Example:

```pony
class X
  new constructor() => None // constructor definition
  
X.constructor() // instance creation, calling the constructor
```

Reply:
Hmm, are new and constructor **always** used together? It feels weird to have both be the same from a non-Pony user's pov